push_dialog.py

```diff
@@ -139,8 +139,7 @@ def show_warning_about_layers_that_cant_work_offline(self, layers):
         QtGui.QMessageBox.information(self.iface.mainWindow(), 'Warning','Layers {} require a real time connection'.format(layers_list))
 
     def set_hybrid_flag(self):
-        #TODO
-        pass
+        QgsProject.instance().writeEntry(QgsOfflineEditing.PROJECT_ENTRY_SCOPE_OFFLINE,"REMOTE_LAYER_MODE",RemoteOptions.HYBRID)
```

Comment:
Actually it is more a QField flag something like

`QgsProject.instance().writeEntry(QFIELD_SCOPE,"REMOTE_LAYER_MODE",RemoteOptions.HYBRID)`